EBH-4350/find_vcfs.py

```
@@ -1,34 +1,85 @@
+        help="Whether to unarchive files."
+    )
+    parser.add_argument(
+        "--output_prefix", type=str, required=False, default="",
+        help="Prefix for output file names."
+    )
+    return parser.parse_args()
+
+
+def find_projects(name: str,
+                  name_mode: str = "regexp",
+                  created_after: str = None,
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)